playbooks/vars/maas-auth.yml

```diff
@@ -0,0 +1,11 @@
+maas_username: "admin"
+
+maas_api_key: "None"
```

Comment:
@briancurtin `maas_api_key` above defaults to `null`, here ti's being set to `"None"`.